components/symbl_ai/actions/get-topics/get-topics.mjs

```diff
@@ -0,0 +1,45 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-topics",
+  name: "Get Topics",
+  description: "Get a list of all the topics generated from the conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/get-topics)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      type: "string",
+      label: "Conversation Id",
+      description: "The Id of the Conversation",
+    },
+    sentiment: {
+      type: "boolean",
+      label: "Sentiment",
+      description: "Provides you sentiment analysis for each topic of the conversation",
+      optional: true,
+      default: false,
+    },
+    parentRefs: {
+      type: "boolean",
+      label: "Topic Hierarchy",
+      description: "Provides you topic hierarchy for each topic of the conversation",
+      optional: true,
+      default: false,
+    },
+  },
+  async run({ $ }) {
+    try {
+      const response = await this.symblAIApp.getTopics({
+        $,
+        conversationId: this.conversationId,
+        params: `sentiment=${this.sentiment}&parentRefs=${this.parentRefs}`,
```

Comment:
As mentioned above, you could pass the query parameters to the `axios` call using axios request config's `params` property:
```js
      const response = await this.symblAIApp.getTopics({
        $,
        conversationId: this.conversationId,
        params: {
          sentiment: this.sentiment,
          parentRefs: this.parentRefs,
        },
      });
```

In `symbl_ai.app.mjs`:
```js
    async getTopics({
      $,
      conversationId,
      params,
    }) {
      return this.makeRequest({
        $,
        path: `/conversations/${conversationId}/topics`,
        params,
      });
    },
```